internal/storage/metrics/metrics.go

```diff
@@ -0,0 +1,35 @@
+package metrics
+
+import (
+	"strconv"
+
+	"github.com/JohnRobertFord/go-plant/internal/config"
+)
+
+type Element struct {
+	ID    string   `json:"id" db:"name"`
+	MType string   `json:"type" db:"type"`
+	Delta *int64   `json:"delta,omitempty" db:"delta"`
+	Value *float64 `json:"value,omitempty" db:"value"`
+}
+
+type Storage interface {
+	Insert(Element) Element
```

Comment:
```suggestion
	Insert(context.Context, Element) (Element, error)
```
И так для всех интерфейсов